website/docs/components/side-nav/partials/code/component-api.md

```diff
@@ -14,6 +14,30 @@ This is the full-fledged component (responsive and animated).
   <C.Property @name="<:footer>" @type="named block">
     A named block where the content for the “footer” section of the SideNav is rendered. It yields the value of `isMinimized` too.
   </C.Property>
+  <C.Property @name="isResponsive" @type="boolean" @default="true">
+    Controls whether the SideNav is responsive to viewport changes. It can be programmatically turned off by passing `false`.
+    <br>
+    <em>Notice: even if the @isResponsive parameter is set to false, some JavaScript is executed anyway in the background, and event listeners are attached to some DOM elements (even if this functionality is not used).</em>
+  </C.Property>
+  <C.Property @name="hasA11yRefocus" @type="boolean" @default="true">
```

Comment:
[question] do we know the ramifications for accessibility if people opt out of this (which I suspect could be common if it's the most effective way to work around the issues folks were reporting). My fear would be that once people opt out of this, getting them to opt back in could be difficult (or impossible).

Reply:
Good point. Probably we should document it. @MelSumner can you add a paragraph or two about this in the PR?

Reply:
[🌶️  question] _should_ we let consumers opt out (in an ideal world)?

Reply:
They may decide to implement their own solution. Or they may be already using another one. Or they may have good reasons for not using it. In any case, it's always easier to have a simple switch, than having them hack something around. _Guidance over control_.

Reply:
Arguably it is easier to see who has this (or any feature for that matter) turned off and potentially work with them to find out why and how we can support them to align with our recommendations, than not providing it and being unable to know who has workarounds.

Reply:
That being said, introducing this argument means that the component may become conditionally conformant – so we should probably flag that in the Accessibility section

Reply:
if `isResponsive` is set to true, then the component will not pass the Reflow Success Criteria.

Reply:
> [🌶️ question] _should_ we let consumers opt out (in an ideal world)?

Alas we are not in an ideal world. We have already set a precedent for a non-conformant escape hatch (see FlightIcon docs) and here we are continuing that precedent. However, we could use stronger language like we did before, that indicates that these API are more like "if you must, while you're transitioning to HDS components and working through your a11y issues at an application level."